test/parallel/test-event-emitter-listeners-side-effects.js

```diff
@@ -21,7 +21,6 @@ assert(fl.length === 1);
 assert(fl[0] === assert.fail);
 
 e.listeners('bar');
-assert(!e._events.hasOwnProperty('bar'));
```

Comment:
Yep, definitely semver major. There are other side effects of this. For instance, there is no `toString` or `valueOf` so doing a console.log or util.inspect will yield different results and many of the `Object.{method}` static methods may croak a bit on it (e.g. Object.keys will still work but I believe some of the others fail).